openbb_platform/core/openbb_core/app/model/obbject.py

```diff
@@ -59,6 +61,8 @@ class OBBject(Tagged, Generic[T]):
         description="Extra info.",
     )
     _credentials: ClassVar[Optional[BaseModel]] = None
```

Comment:
we prolly don't need to have these `credentials`, since we've access to `user_settings` which contain them

Reply:
Yes! You're right - didn't change it yet for the sake of simplicity but if moving forward that's something to change